sig-automation/README.md

```diff
@@ -0,0 +1,137 @@
+# SIG Automation
+**Chair**: John Belamaric, Google, @johnbelamaric
+
+**Vice-chair**: Wim Henderickx, Nokia, @henderiw
+
+## Charter
+
+As agreed to by the TSC and documented in the approved [Nephio Community
+Document](https://docs.google.com/document/d/1agsjCN3aCjgPftO8AL4sCkv3Dp2ZRZ82-ikuVHAJTL8/edit?usp=sharing),
+this SIG group focuses on design and standardization of schemas, CRDs, and operators; development of CRDs, Operators, related tooling & reference Implementation; and packaging and testing of functional components of Nephio.
+
+## Subprojects
+
+Recall the high-level functional components shown in the *Nephio Functional
+Areas* figure.
+
+![Nephio Functional Areas](../Nephio-Functional-Areas.png)
+
+The following subproject structure is defined in the terms of the figure, and
+is intended to faciliate the delivery of R1. We will reconsider this structure
+post-R1 for better longterm alignment with the community needs.
+
+For each subproject, we list a lead and the scope of the subproject, as well as
+the primary skills involved in participating in that subproject. As an open
+community, we do not "require" any specific skills for contributors to
+participate in the subproject. We hope to find ways for anyone to contribute,
+and to build the necessary skills. However, it can be useful to list the primary
+skills and areas of expertise involved in each group, to help people decide
+where they may wish to participate.
+
+### Package Management
+**Subproject Lead**: John Belamaric, @johnbelamaric
+
+**Scope**
+- User Interaction Layer
+- Intent Design
+  - Intent Design
+  - Intent Specialization
+  - Intent Validation
+- Intent Deployment
+  - Cluster Selection
+  - Cross-Cluster Dependencies
+
+In terms of package lifecycle, this includes everything up to and including "fan
+out", where the initial instances of per-cluster packages have been created and
```

Comment:
How about linking the "fanout" terminology to our glossary: https://github.com/nephio-project/docs/blob/main/glossary.md#fanout ?
Same could be applicable to other specific terms introduced by Nephio present in this document.

WDYT?

Reply:
yes, good idea, will do

Reply:
I linked to them in the "Primary Skills" lists but not for every occurrence of the words. Let me know if you see some I missed!